File: src/main/java/no/nav/boot/conditionals/OnGCPCondition.java
package no.nav.boot.conditionals;

import static no.nav.boot.conditionals.Cluster.gcpClusters;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnGCPCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return gcpClusters();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnGCPCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.gcpClusters
import org.springframework.core.type.AnnotatedTypeMetadata

class OnGCPCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = gcpClusters()
}